lighthouse-core/audits/legacy-javascript.js

```diff
@@ -383,6 +383,12 @@ class LegacyJavascript extends Audit {
     ];
     const details = Audit.makeTableDetails(headings, tableRows);
 
+    /** @type {LH.Audit.Details.DebugData} */
+    const debugData = {
+      type: 'debugdata',
+      signalCount,
```

Comment:
do we want any higher fidelity data here @connorjclark ? this just an int.

Reply:
what's our data budget? I was just concerned that there may be a lot of signals.

Reply:
i'm ok with just signal for now, may give some insight via httparchive, and could do manual runs to get more data. so this is fine